.circleci/config.yml

```diff
@@ -17,17 +17,19 @@ jobs:
           name: "Create SFDX Folder" 
           command: |
             # Create sfdx directory
-            mkdir sfdx
+            mkdir ~/sfdx
       - run:
           name: Download CLI
           command: |
             # Download cli
-            wget -qO- $DX_CLI_URL | tar xJ -C sfdx --strip-components 1
+            # By default, the script installs the current version of Salesforce CLI. To install the release candidate, set the DX_CLI_URL_CUSTOM local variable to the appropriate URL
+            wget -qO- ${DX_CLI_URL_CUSTOM-$DX_CLI_URL} | tar xJ -C ~/sfdx --strip-components 1
```

Comment:
All sfdx CLI env vars begin with `SFDX_`.  Is this one deliberately not following that pattern?

Reply:
I remember having having a similar discussion with Juliet, and we determined that `SFDX_` is for CLI use only, therefore I didn't want to create a variable name that would confuse users, this variable only comes into play when we do `wget` but Im open to suggestions Im not too sure about `DX_CLI_URL_CUSTOM`